bindata/network/ovn-kubernetes/002-rbac.yaml

```diff
@@ -22,6 +22,13 @@ rules:
   - list
   - watch
   - patch
+- apiGroups:
```

Comment:
Does ovnkube-node need to list/watch endpointslices? I looked at your PR here: https://github.com/openshift/ovn-kubernetes/pull/364/files and it doesn't seem to be the case.

Reply:
yeah, I was trying to avoid that, but seems that there is no other way that watching endpoints in the node to delete stale conntrack entries.
If we want to delete stale conntrack entries for dual-stack services/endpoints we'll have to watch them in the nodes too, you can see that I'm skipping that test in the PR

Reply:
let's go with the master by now, I think that we can get rid of slices in the nodes :small_airplane:

Reply:
Oh! Well, if it is indeed needed I think you can keep it.

Reply:
I think that is not, need to test it to confirm it, but if we are able to delete the stale conntrack entries by serviceIP instead by endpoints is a big performance improvement too, and avoid using slices of course
https://github.com/ovn-org/ovn-kubernetes/pull/1857/commits/867f1938853dcea6c97f432feafc0d47dbe7b984

Reply:
OK, that is great then!

Reply:
it turns out it does